packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,261 @@
+  const burnAddressPrivateKey = '93ViFmLeJVgKSPxWGQHmSdT5RbeGDtGW4bsiwQM2qnQyucChMqQ'
+  let fundedAddress: string
+  let history: BurnHistory[]
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+
+    await client.wallet.importPrivKey(burnAddressPrivateKey)
+
+    await createHistory()
+
+    history = await client.account.listBurnHistory()
+    // reverse history to test correct order of transaction
+    history.reverse()
+  })
+
+  const createHistory = async (): Promise<void> => {
+    // test masternode creation fee burn
+    const newAddress = await container.getNewAddress('', 'legacy')
+    await client.masternode.createMasternode(newAddress)
+    await container.generate(1)
+
+    // creates funded account address
+    fundedAddress = await container.getNewAddress()
+    await client.wallet.sendToAddress(fundedAddress, 1)
+    await container.call('utxostoaccount', [{ [fundedAddress]: '3@0' }])
+    await container.generate(1)
+
+    // burn token
+    await createToken(container, 'GOLD', { collateralAddress: fundedAddress })
+
+    // mint token and send token to burn address
+    await container.call('minttokens', ['100@GOLD'])
+    await container.generate(1)
+
+    await accountToAccount(container, 'GOLD', 100, {
+      from: fundedAddress, to: burnAddress
+    })
+    await container.generate(1)
+
+    // utxostoaccount burn
+    await container.call('utxostoaccount', [{ [burnAddress]: '1@0' }])
+    await container.generate(1)
+
+    // send to burn address with accountToAccount
+    await accountToAccount(container, '0', 1, {
+      from: fundedAddress, to: burnAddress
+    })
+    await container.generate(1)
+
+    // send to burn address with accounttoutxos
+    const payload: BalanceTransferPayload = {}
+    payload[burnAddress] = '2@0'
+    await client.account.accountToUtxos(fundedAddress, payload)
+    await container.generate(1)
+
+    // send utxo to burn address
+    await client.wallet.sendToAddress(burnAddress, 10)
+    await container.generate(1)
+  }
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listBurnHistory', async () => {
+    expect(history.length).toStrictEqual(7)
+    for (let i = 0; i < history.length; i += 1) {
+      const burnHistory = history[i]
+      expect(typeof burnHistory.owner).toStrictEqual('string')
+      expect(typeof burnHistory.blockHeight).toStrictEqual('number')
+      expect(typeof burnHistory.blockHash).toStrictEqual('string')
+      expect(typeof burnHistory.blockTime).toStrictEqual('number')
+      expect(typeof burnHistory.type).toStrictEqual('string')
+      expect(typeof burnHistory.txn).toStrictEqual('number')
+      expect(typeof burnHistory.txid).toStrictEqual('string')
+      expect(burnHistory.amounts.length).toBeGreaterThan(0)
+      expect(typeof burnHistory.amounts[0]).toStrictEqual('string')
+    }
+  })
```

Comment:
no `let` so move to here
and just need one for general `list` test
```
const history = listBurnHistory()
for (...) {
   // general assertion here (eg: type)
   // blockhash, blocktime, txid, txn

   // distinct assertion here
   // type, owner, amounts
}
```

Reply:
Got it, let me remove duplicate test and assertions